internal/resource/config/password_credential_validator_resource.go

```diff
@@ -32,6 +32,18 @@ var (
 	_ resource.ResourceWithImportState = &passwordCredentialValidatorsResource{}
 )
 
+var (
+	attrType = map[string]attr.Type{
+		"name": basetypes.StringType{},
+	}
+
+	attributeContractTypes = map[string]attr.Type{
```

Comment:
This looks pretty close to the idp adapters resource, except on idp adapters there are a couple booleans in the attribute types. This might be something that could be common as well.